tests/integration/kubernetes/runtimeclass_workloads/busybox-template.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   name: POD_NAME
 spec:
   terminationGracePeriodSeconds: 0
-  runtimeClassName: kata
+  runtimeClassName: RUNTIMECLASS_NAME
```

Comment:
Why do we need this change?
The way we're setting things up in the CI, `kata` is always there and will be valid for our case.

Reply:
Fixed.